types/markdown-it/lib/helpers/parse_link_label.d.ts

```diff
@@ -1,5 +1,7 @@
 import StateInline = require('../rules_inline/state_inline');
 
+declare namespace parseLinkLabel {}
```

Comment:
Did you add this to allow importing like so? `import * as md from "markdown-it"`. This hack will emit bad code for a lot of users, so please don't use it. Instead, individual people need to set `"esModuleInterop": true` and then use `import md from "markdown-it"`. But, for backward compatibility, tests on Definitely Typed will have to use the old `import md = require("markdown-it")` syntax.